### DISCUSSION TITLE
I LOVE GHEALPER, THANK YOU

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1764] [General]
@seerge Hai good sir thank you very much for making this Awsome Tool☺️👌.... I am from India and i really love simple yet powerful software , i wish if i knew about this app 1 year ago so i could have saved myself frome some 
big troubles caused by armoury crate and probably run games without stuttering + overheating on my ROG G15 2021 (HQ403TS-G513QM).I am new to the Github my concern is that when i was exploring the settings of github and stuff i intensionally opened a file of GHELPER  under ryzen section as Githubdev and selected some codes and pressed backspace and i think i accedently deleted them! (Is it possible for some normal people like me can clear or edit codes of your project or any other peoples project like yours?).Does that mean i messed up ghelper source code or file or something? , will it affect something inside ghelper app that gonna available to download and use from now on for the next person......Forgive my foolishness and little knowledge on how the app development works, I don't know how this site works and what is Fork i was also messing with that option?
Google didn't helped me so asking the creator of GHELPER to clear my concern.. !?😣🙏

**Comment by seerge** [comment]
Hello, you don't have permissions to modify anything here :) so if you changed something, you did it only in your own copy

**Reply by ghost** [reply]
That's a relief to know, i actually asked those questions to chatgpt today and gave me same answer xd....Again Thanku For making this awsome simple app, please keep the lightweight aesthetic of the app like this forever ..." If it's perfect why change " XD
